models/decision.go

```diff
@@ -52,6 +52,13 @@ type DecisionWithRuleExecutions struct {
 	RuleExecutions []RuleExecution
 }
 
+type DecisionsByVersionByOutcoume struct {
```

Comment:
```suggestion
type DecisionsByVersionByOutcome struct {
```
small typo here also